test/fonda/core_test.cljs

```diff
@@ -788,4 +798,64 @@
                      (fn [ctx res]
                        (is (= ctx {:steps [:injected-step1 :injected-step2]}))
                        (done))
-                     anomaly-cb-throw))))
\ No newline at end of file
+                     anomaly-cb-throw))))
+
+(deftest callbacks-wrapper-fn-global-on-success
+  (testing "If a callback wrapper is passed, it should be called with the value of the on-callbacks and the arguments
+  that normally would be passsed to the on-callback-fns."
+    (async done
+      (let [processor-res 42
+            on-success-event ::on-success-event
+            processor-path [:success-processor-path]
+            processor {:on-success (cb-with-result ::auth-success)
```

Comment:
How is this passing test? `cb-with-result` returns a function that will be called instead of the provided `:callbacks-wrapper-fn` and `:callbacks-wrapper-fn` you're passing shouldn't get called in this case. Am I missing something here?

Reply:
Yes, that is correct. If you provide a function, it will be called. If you pass a value, then callbacks-wrapper-fn will be called with that value

Reply:
Okay, so this function gets called for global on-* functions as well. Thanks for the clarification :)